server/routes/customer.js

```diff
@@ -82,12 +82,13 @@ customerRouter.post('/register', async (req, res) => {
     .then((customers) => {
       if (customers.length > 0) {
         res.send('customer');
+        console.info('CUSTOMER FOUND', customers[0]);
```

Comment:
remove

Reply:
This will be removed on final sweep tonight after styling